lib/happo/views/diffs.erb

```diff
@@ -10,52 +10,27 @@
     <style>
       <%= Happo::Utils.css_styles %>
     </style>
-  </head>
-
-  <body>
-    <header class="header">
-      <h1 class="header_title">
-        <%= Happo::Utils.page_title(diff_images, new_images) %>
-      </h1>
-      <div class="header__time">Generated: <%= Time.now %></div>
-    </header>
-
-    <main class="main">
-      <% if diff_images.count > 0%>
-        <h2 id="diffs">
-          <a class="anchored" href="#diffs">
-            Diffs (<%= diff_images.count %>)
-          </a>
-        </h2>
 
-        <% diff_images.each do |image| %>
-          <h3 id="<%= Happo::Utils.image_slug(image) %>">
-            <a class="anchored" href="#<%= Happo::Utils.image_slug(image) %>">
-              <%= Rack::Utils.escape_html(image[:description]) %> @ <%= image[:viewport] %>
-            </a>
-          </h3>
+    <script src="https://npmcdn.com/react@15.3.1/dist/react.min.js"></script>
+    <script src="https://npmcdn.com/react-dom@15.3.1/dist/react-dom.min.js"></script>
+    <script src="https://npmcdn.com/babel-core@5.8.38/browser.min.js"></script>
```

Comment:
I am a little worried about what happens if npmcdn is unavailable. It seems like it should be pretty reliable, but I'm pretty sure it is just one person's side-project. 

> I do my best to keep it running, but sometimes things go wrong. Sometimes there are network or provider issues outside my control. Sometimes abusive traffic temporarily affects response times. Sometimes I break things by doing something dumb, but I try not to.

https://npmcdn.com/#/about

I wonder how hard it would be to have a fallback in place.